PSSailpoint/Configuration.ps1

```diff
@@ -265,7 +265,7 @@ function Get-AccessToken {
                     $Data = ConvertFrom-Json $Response.Content
                     $Token = $Data.access_token
                     $TokenExpiration = (Get-Date).AddSeconds($Data.expires_in)
-                    Set-DefaultConfiguration -Token $Token -TokenExpiration $TokenExpiration
+                    Set-DefaultConfiguration -Token $Token -TokenExpiration $TokenExpiration -Proxy $Script:$Configuration["Proxy"]
```

Comment:
Hello @Anneragh, thank you for your contribution!

It looks like you have a syntax error on this line and its causing some failures.

```powershell
 -Proxy $Script:$Configuration["Proxy"]
```

Should be without the `$` on Configuration.

```powershell
 -Proxy $Script:Configuration["Proxy"]
```

Reply:
I removed the typo...